plugin.video.hrmagplayer/libplayer/player.py

```diff
@@ -68,27 +70,37 @@ def list_episodes(context, show):
         # Create a list item with a text label and a thumbnail image.
         list_item = xbmcgui.ListItem(label=episode['title'], thumbnailImage=episode['image'])
         list_item.setArt({"thumb": episode['image']})
+        if 'text' in episode:
```

Comment:
thumbnailImage kwarg above is deprecated, not sure how this passed on the past update. Please drop in favor of list_item.setArt({"thumb":"xxx"})